src/analysis/functions.rs

```diff
@@ -60,6 +63,12 @@ fn analyze_function(env: &Env, func: &library::Function, class_tid: library::Typ
         }
     }
 
+    if !commented {
+        for s in used_types {
+            all_used_types.insert(s);
+        }
+    }
```

Comment:
This  might need to be revisited when we encounter unused import warnings depending on the set cfg versions.

An alternative is putting something like this in modules that have version cfgs:
```
#![cfg_attr(not(gtk_3_10), allow(unused_imports))]
```